acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -16,8 +16,8 @@ def initialize_repo_on_host(host, os, nightly)
         on host, "dpkg -i puppet5-nightly-release-$(lsb_release -sc).deb"
 
       else
-        on host, "curl -O http://apt.puppetlabs.com/puppetlabs-release-pc1-$(lsb_release -sc).deb"
-        on host, "dpkg -i puppetlabs-release-pc1-$(lsb_release -sc).deb"
+        on host, "curl -O http://apt.puppetlabs.com/puppet-release-$(lsb_release -sc).deb"
+        on host, "dpkg -i puppet-release-$(lsb_release -sc).deb"
```

Comment:
So, this enables the new rolling repos, which are linked to the current major series out (currently puppet5). You can definitely pin to a major series, but if you set it here, these puppet repos will be updated every time we switch over to a new series. So, we'll update the link to point to puppet6 when the time comes. I'm not sure if you want this behavior or not, so I figured I'd point it out.